### DISCUSSION TITLE
Dask Dashboard URL for LocalCluster

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [nebari-dev/nebari#1572] [Show and tell]
I was happy to figure out how to display the Dask Dashboard when you have a `LocalCluster` (usually used just to show how limited it is compared to the Dask Gateway cluster, but still...). 

Just take your username and the port number reported by `LocalCluster` and create a URL like this:
`https://nebari.esipfed.org/user/{user}/proxy/{port}/status`

So if my username is 'rsignell-usgs' and the repoted daskboard is http://127.0.0.1:33723/status, then the link will be:
https://nebari.esipfed.org/user/rsignell-usgs/proxy/33723/status

Like this:

![2022-10-26_11-20-10](https://user-images.githubusercontent.com/1872600/198068461-0dc6d035-2a07-483e-af9e-78a7ab1b9a78.png)

**Comment by dharhas** [comment]
I have some code you can use to rewrite the URL that is shown to be the correct one. Let me find the snippet.

**Comment by dharhas** [comment]
this issue has a comment that shows how you can override the link displayed by using a setting in dasks config. not sure if you can set different things for localcluster and dask-gateway though.

https://github.com/nebari-dev/nebari/issues/84
